web_widget_switchcase/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web Switchcase Widget",
+    "version": "10.0.1.0.0",
+    "author": "Agile Business Group, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Web",
+    "website": "https://github.com/OCA/web/tree/10.0/web_widget_switchcase",
+    'installable': True,
+    "depends": [
+        "web",
+    ],
+    "data": [
+        "views/web_widget_switchcase.xml",
+    ],
+    "js": [
+        "static/src/js/web_widget_switchcase.js",
```

Comment:
Are the `js` and `test` keys required? It's already included in web_widget_switchcase/views/web_widget_switchcase.xml